### PR TITLE
fixes for R1 SDF https://github.com/PX4/sitl_gazebo/issues/533

### PR DESCRIPTION
**Pull request by bperseghetti**
Fixes for R1 that also requires PR for Firmware: https://github.com/PX4/Firmware/pull/15209  to work fully.
Fixes issues brought up in https://github.com/PX4/sitl_gazebo/issues/533

### CODE REVIEW

File: models/r1_rover/r1_rover.sdf [hunk 1]
```diff
@@ -1,24 +1,24 @@
-<sdf version='1.6'>
+<sdf version='1.7'>
```
Comment (Jaeyoung-Lim) [inline models/r1_rover/r1_rover.sdf]: This breaks gazebo9, since it is not a supported sdf version. Why was this necessary?

File: models/r1_rover/r1_rover.sdf [hunk 2]
```diff
@@ -131,39 +131,51 @@
...
+          <bounce>
+            <restitution_coefficient>0</restitution_coefficient>
+            <threshold>1e6</threshold>
+          </bounce>          
           <contact>
             <ode>
-              <min_depth>0.005</min_depth>
-              <kp>1e8</kp>
+              <min_depth>0.001</min_depth>
+              <mav_vel>0.0</mav_vel>
```
Comment (Jaeyoung-Lim) [inline models/r1_rover/r1_rover.sdf]: This results in a failure when running on gazebo 9.13.0 
```
Warning [parser.cc:950] XML Element[mav_vel], child of element[ode] not defined in SDF. Ignoring[mav_vel]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
Warning [parser.cc:950] XML Element[mav_vel], child of element[ode] not defined in SDF. Ignoring[mav_vel]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
Warning [parser.cc:950] XML Element[mav_vel], child of element[ode] not defined in SDF. Ignoring[mav_vel]. You may have an incorrect SDF file, or an sdformat version that doesn't support this element.
```

**Review by Jaeyoung-Lim — CHANGES_REQUESTED** [review]

File: models/r1_rover/r1_rover.sdf (hunk 2 above)
Comment (bperseghetti) [reply models/r1_rover/r1_rover.sdf]: @Jaeyoung-Lim thanks, I just fixed the typo in the ode (max_vel) and made it so people still stuck to gazebo 9 can use it by bumping it down to 1.6 ;-) . https://github.com/PX4/sitl_gazebo/pull/534/commits/7f18e8724c82c8e58dbb7c95adaefd077950b140

**Comment by Jaeyoung-Lim** [comment]
@bperseghetti Thanks for getting this working! How are you testing this?

While testing this PR, I noticed that the gps position is not valid in this model anymore, probably because https://github.com/PX4/sitl_gazebo/pull/517 wasn't ported to this model probably. Are you only testing in manual mode with the joysticks?

**Comment by TSC21** [comment]
> @bperseghetti Thanks for getting this working! How are you testing this?
> 
> While testing this PR, I noticed that the gps position is not valid in this model anymore, probably because #517 wasn't ported to this model probably. Are you only testing in manual mode with the joysticks?

Yes it was https://github.com/PX4/sitl_gazebo/pull/517/files

**Comment by Jaeyoung-Lim** [comment]
@TSC21 Ah, sorry. @bperseghetti I guess this means this PR needs a rebase on the latest master

**Comment by bperseghetti** [comment]
@Jaeyoung-Lim @TSC21 here: https://github.com/PX4/sitl_gazebo/pull/534/commits/34fe1312ce722229056ab9deb0c31380fdd13f91

**Review by Jaeyoung-Lim — APPROVED** [review]
Thanks! Tested locally, and is a huge improvement in controls!

Mission mode is still not working, but should be fixed on the firmware side. 

The sdf validation is failing on https://travis-ci.org/github/PX4/sitl_gazebo/jobs/701776596#L234

**Comment by Jaeyoung-Lim** [comment]
Fixes #533 

@TSC21 What would be your take on the sdf schema validation? It seems that the sdf schema isn't being updated anymore
